flutter/lib/src/default_integrations.dart

```diff
@@ -194,9 +194,12 @@ class WidgetsBindingIntegration extends Integration<SentryFlutterOptions> {
 
     // We don't need to call `WidgetsFlutterBinding.ensureInitialized()`
     // because `FlutterSentry.init` already calls it.
-    WidgetsBinding.instance.addObserver(_observer);
-
-    options.sdk.addIntegration('widgetsBindingIntegration');
+    // If the instance is not created, we skip it to keep going.
+    final instance = WidgetsBinding.instance;
+    if (instance != null) {
+      instance.addObserver(_observer);
```

Comment:
Shouldn't that be logged to the console?

Reply:
good catch, added an else branch that calls `options.logger`